pkg/ctl/cmdutils/nodegroup_flags.go

```diff
@@ -48,6 +48,8 @@ func AddCommonCreateNodeGroupFlags(cmd *cobra.Command, fs *pflag.FlagSet, p *api
 
 	fs.StringToStringVar(&ng.Labels, "node-labels", nil, `Extra labels to add when registering the nodes in the nodegroup, e.g. "partition=backend,nodeclass=hugememory"`)
 	fs.StringSliceVar(&ng.AvailabilityZones, "node-zones", nil, "(inherited from the cluster if unspecified)")
+
+	fs.StringSliceVar(&ng.TargetGroupARNs, "node-target-groups", []string{}, "Attach target groups to the nodegroup")
```

Comment:
We tend to avoid adding new flags, unless it's something that is very commonly used. Would you mind to remove the flag and keep only for the config file please?

Reply:
Okay, I'll remove it.

Reply:
Thanks! Just ping when it's done :)